examples/example3/example3_withmanufactured.ipynb

```diff
@@ -0,0 +1,679 @@
+    "By = (k_kin*ySize/(2*D)) / ((yFactor/2)*(np.exp(yFactor)-1) + (k_kin*ySize/(2*D))*(1+np.exp(yFactor)))\n",
+    "zFactor = zSize / np.sqrt(3*D/k_p)\n",
+    "Az = (k_kin*zSize/(2*D))*np.exp(zFactor) / ((zFactor/2)*(np.exp(zFactor)-1) + (k_kin*zSize/(2*D))*(1+np.exp(zFactor)))\n",
+    "Bz = (k_kin*zSize/(2*D)) / ((zFactor/2)*(np.exp(zFactor)-1) + (k_kin*zSize/(2*D))*(1+np.exp(zFactor)))\n",
+    "expFactor = np.sqrt(k_p/(3*D))\n",
+    "Asol = cT * ((Ax*sym.exp(-expFactor*x) + Bx*sym.exp(expFactor*x)) * \n",
+    "                (Ay*sym.exp(-expFactor*y) + By*sym.exp(expFactor*y)) * \n",
+    "                (Az*sym.exp(-expFactor*z) + Bz*sym.exp(expFactor*z)) *\n",
+    "                (1 - sym.exp(-k_kin*t)))\n",
+    "f = sym.diff(Asol, t) - D*(sym.diff(sym.diff(Asol,x),x) + sym.diff(sym.diff(Asol,y),y) + sym.diff(sym.diff(Asol,z),z)) + k_p*Asol\n",
+    "f_parameter = Parameter.from_expression(\"f_parameter\", str(f), uM/sec)#, use_preintegration=True)\n",
+    "f_reaction = Reaction(\"f_reaction\", [], [\"Aphos\"], param_map={\"k\": \"f_parameter\"}, species_map={}, eqn_f_str=\"k\")\n",
+    "pc.add([f_parameter])\n",
+    "rc.add([f_reaction])\n",
+    "\n",
+    "gleft = -D*sym.diff(Asol,x) - k_kin*(Atot.value-Asol)\n",
+    "gleft.subs(x, 0.0)\n",
+    "gright = D*sym.diff(Asol,x) - k_kin*(Atot.value-Asol)\n",
+    "gright.subs(x, xSize)\n",
+    "gfront = -D*sym.diff(Asol,y) - k_kin*(Atot.value-Asol)\n",
+    "gfront.subs(y, 0.0)\n",
+    "gback = D*sym.diff(Asol,y) - k_kin*(Atot.value-Asol)\n",
+    "gback.subs(y, ySize)\n",
+    "gbottom = -D*sym.diff(Asol,z) - k_kin*(Atot.value-Asol)\n",
+    "gbottom.subs(z, 0.0)\n",
+    "gtop = D*sym.diff(Asol,z) - k_kin*(Atot.value-Asol)\n",
+    "gtop.subs(z, zSize)\n",
+    "\n",
+    "g_params = []\n",
+    "g_reactions = []\n",
+    "PMsub_list = [\"PMleft\", \"PMright\", \"PMfront\", \"PMback\", \"PMbottom\", \"PMtop\"]\n",
+    "gExpr_list = [gleft,  gright,  gfront,  gback,  gbottom,  gtop]\n",
+    "for i in range(len(PMsub_list)):\n",
+    "    g_params.append(Parameter.from_expression(f\"gParam_{PMsub_list[i]}\", gExpr_list[i], uM*um/sec))#, use_preintegration=True))\n",
+    "    g_reactions.append(Reaction(f\"g_{PMsub_list[i]}\", [], [\"Aphos\"], param_map={\"k\": f\"gParam_{PMsub_list[i]}\"}, \n",
+    "                                species_map={}, eqn_f_str=\"k\", explicit_restriction_to_domain=PMsub_list[i]))\n",
+    "pc.add(g_params)\n",
+    "rc.add(g_reactions)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "b432d407",
+   "metadata": {},
+   "source": [
+    "## Initialize model and solver\n",
+    "\n",
+    "Now we modify the solver configuration for this problem. In the solver config, we set the final t as 1 s, the initial dt at .01 s (without any additional specifications, this will be the time step for the whole simulation), and the time precision (number of digits after the decimal point to round to) as 6."
```

Comment:
```suggestion
    "Now we modify the solver configuration for this problem. In the solver config, we set the simulation end time (`\"final_t\"`) as 1 s, the initial time step (`\"initial_dt\"`) at .01 s (without any additional specifications, this will be the time step for the whole simulation), and the time precision (number of digits after the decimal point to round to) as 6."
```